metricshub-agent/src/main/dist/config/metricshub-rules.yaml

```diff
@@ -0,0 +1,63 @@
+groups:
+
+- name: MetricsHub-Host-Availability
+  rules:
+  - alert: MetricsHub-Host-Unreachable-Critical
+    expr: absent(metricshub_host_up)
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: >
+        {{ $labels.host_name }} — Host is unreachable
+      description: |
+        No monitored protocol has reported from host {{ $labels.host_name }} for at least 5 minutes.
+        The host or network path is likely down.
+        Consequence: The host monitoring is probably interrupted and no metrics are being collected.
+        Recommended Action: Verify the status of the protocols configured to monitor the host. Ensure the host is running and reachable.
+
+- name: MetricsHub-Connector-Status
+  rules:
+  - alert: MetricsHub-Connector-Failed-Critical
+    expr: metricshub_connector_status{state="failed"} == 1
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: >
+        {{ $labels.host_name }} — Connector {{ $labels.name }} has failed
+      description: |
+        The connector {{ $labels.name }} on host {{ $labels.host_name }} has been in a failed state for at least 5 minutes.
+        Consequence: This connector is no longer reporting its expected metrics.
+        Recommended Action: Check the status of the protocol used by the connector, review the logs.
+
+- name: MetricsHub-Protocol-Availability
+  rules:
+  - alert: MetricsHub-Protocol-Down-Critical
+    expr: metricshub_host_up == 0
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: >
+        {{ $labels.host_name }} — Protocol {{ $labels.protocol }} is down
+      description: |
+        The {{ $labels.protocol }} protocol on host {{ $labels.host_name }} has been reporting a down status  for at least 5 minutes.
```

Comment:
There is a double space in this sentence (between "status" and "for"):

 The {{ $labels.protocol }} protocol on host {{ $labels.host_name }} has been reporting a down status  for at least 5 minutes.